profiles/fscloud/variables.tf

```diff
@@ -0,0 +1,153 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+variable "resource_group_id" {
+  description = "ID of resource group to use when creating the mongodb database"
+  type        = string
+}
+
+variable "mongodb_version" {
+  description = "The version of mongodb. If null, the current default ICD mongodb version is used."
+  type        = string
+  default     = null
+}
+
+variable "plan" {
+  type        = string
+  description = "The name of the service plan that you choose for your mongodb instance"
+  default     = "standard"
+}
+
+variable "region" {
+  description = "The region mongodb is to be created on. The region must support BYOK ( us-south, us-east, and eu-de)"
+  type        = string
+  default     = "us-south"
+  validation {
+    condition = anytrue([
+      var.region == "us-south",
+      var.region == "us-east",
+      var.region == "eu-de"
+    ])
+    error_message = "region must be in a BYOK location, us-south, us-east or eu-de"
+  }
+}
+
+variable "allowlist" {
```

Comment:
I'm not sure this should exist in the fscloud profile. We want people to use CBRs instead right @daniel-butler-irl ?

Reply:
Allowlist or CBR is acceptable for fscloud but must be one or both